bin/hermit

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+#
+# THIS FILE IS GENERATED; DO NOT MODIFY
```

Comment:
changes from running `hermit init`. Let me know if I should remove

Reply:
Nope this is fine!